### ISSUE TITLE
Content duplicated in non-edit mode

### ISSUE THREAD

**Issue by webstroke** [iseulde/wp-front-end-editor#209]
Dear wpfee team,

Although the editing functionality of your plugin works fantastically for me, it duplicates the content on the page when in non-edit mode, as demonstrated by the attached screenshot. Once in edit mode, everything works perfectly.

Is this a known problem, or if not, what would you like me to provide to help you rectify the issue?

I have attached screenshots to assist.

Kind regards,
Jamie

---

Not in edit mode - weird:
![screenshot 2015-01-07 12 05 39](https://cloud.githubusercontent.com/assets/3436833/5645315/f8d2e85a-9665-11e4-8348-63cef211e56e.png)

In edit mode - normal:
![screenshot 2015-01-07 12 08 22](https://cloud.githubusercontent.com/assets/3436833/5645314/f8d280a4-9665-11e4-9947-46f2eb2184f9.png)

**Comment by Krielkip** [comment]
Looks like you are missing some CSS code. 

By default FEE content is hidden.

**Comment by wellspun** [comment]
I am having the same issue.  I checked whether the CSS is missing and all the CSS files listed in wp-front-end-editor/src/css/ are correctly linked.  I am running my content in a custom theme, however that theme is closely based off of the WP TwentyFifteen theme.  Any ideas?

Thanks!

**Comment by thisispad** [comment]
I also have this issue. 

It is fine on the default theme but not my own. I have no idea what could be causing it. I have disabled my functions.php content, no difference; I have disabled all other js, no difference. No errors in the console. 

When I open a new post, it shows the body content twice - one editable, and one non-editable. Pressing edit post removes it and saving keeps it gone, until refresh when the duplication occurs again.
